guides/persistence/slick.md

```diff
@@ -66,15 +71,15 @@ For this guide we choose the [H2 Database](http://www.h2database.com/html/main.h
 ```scala
 
 libraryDependencies ++= Seq(
-  "com.typesafe.slick" %% "slick" % "2.1.0",
-  "com.h2database" % "h2" % "1.3.166"
+  "com.typesafe.slick" %% "slick" % "3.0.0-RC1",
+  "com.h2database" % "h2" % "1.4.181"
```

Comment:
This is very trivial but latest is 1.4.186. http://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22com.h2database%22%20AND%20a%3A%22h2%22